packages/bpk-component-modal/src/BpkModalV2/BpKModal-test.tsx

```diff
@@ -91,15 +91,37 @@ describe('BpkModalV2', () => {
       expect(document.getElementById('bpk-modal-element')).toBeInTheDocument();
     });
 
-    it('should not render title if title is set to empty string', () => {
+    it('should render title correctly', () => {
       render(
-        <BpkModalV2 {...props} title="">
+        <BpkModalV2 {...props}>
+          <div>Content</div>
+        </BpkModalV2>,
+      );
+
+      expect(screen.getByText('Modal with dialog element')).toBeInTheDocument();
+      expect(screen.getByTitle('bpk-modal-button-close')).toBeInTheDocument();
+    });
+
+    it('should not render title when title does not exist', () => {
+      render(
+        <BpkModalV2 {...props}>
           <div>Content</div>
         </BpkModalV2>,
       );
 
-      expect(screen.queryByRole('Dialog Element')).not.toBeInTheDocument();
-      expect(document.getElementsByClassName('bpk-close-button')).toBeTruthy();
+      expect(screen.queryByText('MyTitle')).not.toBeInTheDocument();
+      expect(screen.getByTitle('bpk-modal-button-close')).toBeInTheDocument();
+    });
```

Comment:
I am not too sure about this test as you are already passing it in via the `props` object but the test just checks if a string that is not in the rendered output, but a title would still be there as the prop with a string exists.

Could you check this one out?

Reply:
Oh, I messed up this test and the previous one which checks that the title is rendered. I'll change both to reflect better what is really happening 👍